docs/content/manual/release-specific/v1.9.0/test-crd-v1beta2-migration.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Test Migrate v1beta1 To v1beta2
+---
+
+## Related issue
+https://github.com/longhorn/longhorn/issues/10250
+
+## Test Steps
+
+**Given** A single-server K3s cluster
+> Ref: https://docs.k3s.io/architecture?_highlight=single&_highlight=server#single-server-setup-with-an-embedded-db
+> Example: https://github.com/k3s-io/vagrant-k3s
+
+**And** Install necessary command line tools on master node
+```bash
+sudo zypper in jq sqlite3
+sudo snap install kubectl --classic
+```
+
+**And** Longhorn v1.2.x installed
+
+**And** Create a RWX volume
```

Comment:
Also need to add `backingImage', 'backingimagedatasource' and 'backingimagemanager'